js/script.js

```diff
@@ -0,0 +1,11 @@
+const menuButtonOpen = document.getElementById('mobile-menu');
+const menuButtonClose = document.getElementById('mobile-menu-exit');
+const navListMenu = document.getElementById('nav-list-menu');
+
+menuButtonOpen.addEventListener('click', () => {
+  navListMenu.classList.add('open');
+});
+
+menuButtonClose.addEventListener('click', () => {
+  navListMenu.classList.remove('open');
+});
```

Comment:
You're doing great in implementing JavaScript on opening and closing the mobile menu, but there are a few changes you need to work on for your project to be completed

- [ ] Kindly make the mobile menu closes when a menu option is clicked. Currently, it's taking the user to the right spot when clicked but the mobile menu still stays in the view. Please fix that so it closes when any of the mobile options is clicked.
- From the design, you can tell it's displaying the about section but the mobile menu still stays in view.

![image](https://user-images.githubusercontent.com/50941074/161813617-591b4b5e-4015-4568-ac5b-4fea1fbfb770.png)